src/otx/algorithms/detection/adapters/openvino/task.py

```diff
@@ -446,14 +446,17 @@ def load_inferencer(
             ):
                 args.append({"resize_type": "fit_to_window_letterbox", "pad_value": 114})
             inferencer: BaseInferencerWithConverter = OpenVINODetectionInferencer(*args)
-        if self.task_type == TaskType.INSTANCE_SEGMENTATION:
-            if self.config.tiling_parameters.enable_tiling:
+        if self.task_type == TaskType.INSTANCE_SEGMENTATION or self.task_type == TaskType.ROTATED_DETECTION:
+            if not self.config.tiling_parameters.enable_tiling:
                 args.append({"resize_type": "standard"})
             else:
                 args.append({"resize_type": "fit_to_window_letterbox", "pad_value": 0})
```

Comment:
Similar as above, MaskRCNN_EfficientNetB2B has keep_ratio w/ False. Is it ok?

Reply:
Thanks, I skipped that as well.